packages/samples/specs/misc/appconfig/api.json

```diff
@@ -185,8 +185,7 @@
                 "locked",
                 "etag"
               ]
-            },
-            "collectionFormat": "csv"
+            }
```

Comment:
for autorest we might want to keep how it was generating before

Reply:
To minimize spec impact?